### ISSUE TITLE
Suggestion: Alliance Chat

### ISSUE THREAD

**Issue by LlmDl** [TownyAdvanced/Towny#926]
Originally reported on Google Code with ID 893

```
I play on a large server with the least amount of players being at 30. I own a Nation,
and am allied to 2 nations, and would love to talk to all allied nations at once, being
able to issue alliance-wide objectives and what-not. So, I was thinking that Alliance
chat could possibly be implemented in the next update or so. 

The command could be something like /ac for Alliance chat. 

Just a suggestion I had while playing on a server.
```

Reported by `VoidFusionVG` on 2012-04-08 21:23:51

**Comment by Drc-DEV** [comment]
Any progress on this?

**Comment by LlmDl** [comment]
Done as of TC 0.62, which will be released after Towny 0.95.0.5 is released.
